components/AddressButtons/AddressButtons.tsx

```diff
@@ -12,39 +14,56 @@ const AddressButtons: React.FC<Props> = ({
   address,
   showInspectButton = true,
 }) => {
+  const { network } = useContext(NetworkContext);
+
+  const networkType = network.name === 'MAINNET' ? 'mainnet' : 'testnet';
+
+  const openProfileExplorer = () => {
+    window.open(
+      `https://wallet.universalprofile.cloud/${address}?network=${networkType}`,
+      '_blank',
+    );
+  };
+
+  const openAssetExplorer = () => {
+    window.open(
+      `https://wallet.universalprofile.cloud/asset/${address}?network=${networkType}`,
+      '_blank',
+    );
+  };
+
+  const openBlockscout = () => {
+    const url =
+      network.name === 'MAINNET'
+        ? `https://explorer.execution.mainnet.lukso.network/address/${address}`
+        : `https://explorer.execution.testnet.lukso.network/address/${address}`;
+    window.open(url, '_blank');
+  };
+
+  const openInspector = () => {
+    window.location.href = `${
+      window.location.href.split('?')[0]
+    }?address=${address}`;
+  };
+
   return (
     <div className="buttons is-centered are-small pt-2">
-      <a
-        className="button is-primary is-light"
-        target="_blank"
-        rel="noreferrer"
-        href={`https://universalprofile.cloud/${address}`}
-      >
-        View on UP as Profile 🧑‍🎤
-      </a>
-      <a
-        className="button is-info is-light"
-        target="_blank"
-        rel="noreferrer"
-        href={`https://universalprofile.cloud/asset/${address}`}
-      >
-        View on UP as Asset 👗
-      </a>
-      <a
+      <button
```

Comment:
I would prefer to keep using `a` instead of `button`. Let's not build fake buttons that are just opening links - it is not good for html accessibility.

- https://www.w3schools.com/accessibility/accessibility_buttons_links.php
- https://a11y-101.com/design/button-vs-link